project/wait_for_postgres.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# wait-for-postgres.sh
```

Comment:
I'd rename this file. `wait-for-postgres.py` wait for pg, but this one could be called `init-postgres.sh` or something like that?

Reply:
Sounds good!